src/services/users.js

```diff
@@ -0,0 +1,77 @@
+const jwt = require('jsonwebtoken')
+const { 
+	genPasswordHash, 
+	verifyPassword
+} = require('../lib')
+
+const config = require('../config')
+const db = require('../db')
+
+const createNewUser = async(user) => {
+	const [
+		firstName, 
+		lastName, 
+		email, 
+		password, 
+		gender, 
+		jobRole, 
+		department, 
+		address 
+	] = user
+	const passwordHash =  genPasswordHash(password)
```

Comment:
@jakazzy I saw this is what was causing the passwordHash to be an empty object

Its a `Promise` and should be awaited but it isn't so the insert happens before the hash finishes generating

Reply:
@jocrah  done